screens/auth/SignUpScreen.js

```diff
@@ -129,7 +131,14 @@ export default class SignUpScreen extends React.Component {
         points: 0,
         pushTokenIds: pushTokenId ? [pushTokenId] : null,
       });
-
+      Sentry.configureScope(scope => {
+        scope.setExtra('setUserScreen', 'SignUpScreen');
```

Comment:
let's remove this -- this will be set and reported when an error is caught.

When we call `configureScope`, any changes we add will be kept on successive calls to `configureScope` so, for example if we kept the call here `scope.setExtra` and there was an error in a different screen, our Sentry log would report this extra info "setUserScreen: signUpScreen" when an error actually occured in rewards.

so let's remove this line, I'll add comments on changes to the logUtils.js function calls